SampleApp/SampleApp.iOS/Entitlements.plist

```diff
@@ -4,5 +4,9 @@
 <dict>
 	<key>aps-environment</key>
 	<string>development</string>
+	<key>keychain-access-groups</key>
+	<array>
+		<string>com.urbanairship.richpush</string>
```

Comment:
this is to work around an issue where xamarin is unable to save to the keychain on the simulator. not an issue on real devices.